### ISSUE TITLE
[Bug] [1.8.01]: Export option menu is blacked out

### ISSUE THREAD

**Issue by FarallonWest** [KosmosisDire/obsidian-webpage-export#520]
### What happened?

I am excited to try this awesome plugin! I select export entire vault and the modal menu pops up but it is blank, empty and has no activity no matter where it is clicked. I have tried light/dark switching. I tried 5 different themes in case there was a conflict there.
<img width="965" alt="Screenshot 2024-09-27 at 2 52 57 PM" src="https://github.com/user-attachments/assets/815d01ec-91bb-4188-a46f-3bc25c1ec9cb">

### Reproduction

1. Install plugin
2. Set settings in plugin menu
3. Go to Command Pallet
4. Select either export with past or export (I tried both)
5. Watch the blank menu pop up

### Last Working Version

Unknown

### Version

1.8.01

### Operating System

Sonoma 14.5

### Obsidian Version

Version 1.6.7 (Installer 1.5.12)

### Relevant log output

```yaml
Log:


Settings:
	settingsVersion --------- 1.8.01
	makeOfflineCompatible     true
	inlineAssets ------------ false
	includePluginCSS          12 plugins included
	includeSvelteCSS -------- true
	titleProperty             title
	customHeadContentPath ---
	faviconPath              
	documentWidth ----------- 40em
	sidebarWidth              20em
	minOutlineCollapse ------ 2
	startOutlineCollapsed     false
	allowFoldingHeadings ---- true
	allowFoldingLists         true
	allowResizingSidebars --- true
	logLevel                  warning
	minifyHTML -------------- true
	makeNamesWebStyle         true
	onlyExportModified ------ true
	deleteOldFiles            false
	addThemeToggle ---------- true
	addOutline                true
	addFileNav -------------- true
	addSearchBar              true
	addGraphView ------------ true
	addTitle                  true
	addRSSFeed -------------- true
	siteURL                  
	authorName -------------- Farallon
	vaultTitle                Guided-Vault
	exportPreset ------------ website
	openAfterExport           false
	graphAttractionForce ---- 1
	graphLinkLength           12
	graphRepulsionForce ----- 114
	graphCentralForce         1.9500000000000002
	graphEdgePruning -------- 100
	graphMinNodeSize          3
	graphMaxNodeSize -------- 7
	showDefaultTreeIcons      true
	emojiStyle -------------- Native
	defaultFileIcon           lucide//file
	defaultFolderIcon ------- lucide//folder
	defaultMediaIcon          lucide//file-image
	exportPath --------------
	filesToExport             0


Enabled Plugins:
	Natural Language Dates
	Zoottelkeeper Plugin
	Note Refactor
	Dataview
	Map View
	Graph Analysis
	Find orphaned files and broken links
	Linter
	Omnisearch
	BRAT
	Import Into Obsidian
	NLP
	Juggl
	Regex Find/Replace
	Slurp
	Charts View
	Link Exploder
	Semantic Canvas
	Text Extractor
	Smart Connections Visualizer
	Smart Connections
	Wikidata Importer
	Graph Link Types
	Wikipedia
	Pandoc Plugin
	Entity Linker
	Footlinks
	Import Attachments+
	Local images
	Surfing
	Wikipedia Search
	Airgap
	Vault 2 Book
	Webpage HTML Export
```

### Additional Info

Included screenshot above.

**Comment by KosmosisDire** [comment]
Hi! Thank you, this should be fixed in the beta version of the plugin. There are instructions for installing the beta version in the readme :)

I am actually working on the next stable release as we speak so if the beta version doens't work I am happy to look into this further to get it working

Thanks

**Comment by KosmosisDire** [comment]
> The modal now shows up and export is super fast! Thanks! But… I created an offline version and there is no TOC on the side to navigate. Further graphs are not rendering — as though the pages are no longer “speaking” to each other.

Did you use local website or raw documents? The graph view is not supported right now in local exports only in online exports (although this will hcange in the next update).
The file navigation is not supported in raw documents exports only in local website and online website exports. The TOC should work in all the modes.

If you could give a screenshot showing which parts aren't working that would be helpful.
Thanks

**Comment by KosmosisDire** [comment]
So it looks like the table of contents is there. I think you mean the vault navigation, right? Is it enabled in the settings? I can't remember if I have it implemented properly in the current beta, but I know that it works in the dev branch in which case it will also be out in the next release.

**Comment by KosmosisDire** [comment]
ETA on next release: hopefully tomorrow!? No promises though lol

Hmm I'm not sure I totally understand what is happening using just the screenshots. And I haven't seen anything like this happen before. Could you possibly take a screen recording of the whole process that you are taking to try and make an export that shows both of the failures you mentioned here? I think that will make it a lot easier to understand what I might be able to do to reproduce your problem and to understand what your problem is. 

Thanks!

**Comment by KosmosisDire** [comment]
Thanks for the video!
A few more questions:
Can you reproduce this in a sandbox vault with no other plugins and maybe one or two files?
How many files total are you trying to export? 
And you are using the local website export from beta version 1.9.0-2b which exports a single file?

**Comment by KosmosisDire** [comment]
Thanks, let me know how the sandbox goes. I am still working on the next beta but it is getting very close so hopefully you can test that out as well soon.

Is it possible for you to also give me a screenshot of the developer console in obsidian during this process so I can see if any errors occur. You can open the developer console with `Option + ⌘ + i` on mac. 

Thanks for you patience and for all the useful information :)

**Comment by KosmosisDire** [comment]
Hi you need to use the beta version to access the local export as that was not a feature before 1.9 :)

**Comment by KosmosisDire** [comment]
Yeah the graph view doesn't work yet in 1.9.0-2b in local exports. However the missing pdf isn't great.

I am actually gonna give you a build of the new beta right now to see if it works there lol (this also means you get a peek at all the new features before anyone else knows about them 😉).  I just tested it on a pretty large complex vault with lots of plugins on the local export.

If you go to the community plugins list and open the plugins folder you will find the folder on your system called webpage-html-export, unzip this file and replace the contents of that folder. Make sure to restart obsidian (You can also delete the `data.json` file in that folder).


[webpage-html-export.zip](https://github.com/user-attachments/files/17220323/webpage-html-export.zip)
🤞Fingers crossed

**Comment by KosmosisDire** [comment]
But it does look like the freezing issue did not happen, so it is possibly an issue with the number of documents you are trying to export or some plugin. I have greatly improved the performance of the feature in the build I sent you above, so go ahead and try that on your main vault too and see if it works any better.

**Comment by KosmosisDire** [comment]
There is actually a button in obsidian itself to open the folder for you. Go to your installed mods list in obsidian and there should be a little folder icon at the top of the list. Click on it to open the folder

**Comment by KosmosisDire** [comment]
Did you enable it? Also check the console for errors

**Comment by KosmosisDire** [comment]
Oh so the plug-in is loaded properly but it doesn't show in the ribbon? Can you send a screenshot of the ribbon and the command pallet? When I say the console could you press Opt+⌘+i and screenshot any errors there? Thanks.

**Comment by KosmosisDire** [comment]
> The new features look awesome!
Oh did it work?

**Comment by KosmosisDire** [comment]
Oops option-cmd-i should be correct. Sorry I don't use Mac haha.

**Comment by KosmosisDire** [comment]
Damnit google has failed me!

**Comment by KosmosisDire** [comment]
Ah okay I see the issue I'll send you a new version lol, you can also just use a different theme. Looks like there is just a syntax error in the theme you are using. But I'll send you one that will work with this theme

**Comment by KosmosisDire** [comment]
[webpage-html-export.zip](https://github.com/user-attachments/files/17220871/webpage-html-export.zip)

Tested this with your theme and it is able to load properly now.

**Comment by KosmosisDire** [comment]
Can you try it with fewer files selected, like maybe 20 or so?

**Comment by KosmosisDire** [comment]
Awesome! So glad it is working for you! You can also try hovering over links to other notes that exist on your export that's the main secret feature haha

**Comment by KosmosisDire** [comment]
What are the warnings?
Does it still work in a sandbox vault?
Can you try with fewer files
And also you could try uninstalling and reinstalling

Not really sure what could be causing this tbh haha

**Comment by KosmosisDire** [comment]
The cache should have no effect if you're using local export. I should probably make it so it is doable if it doesn't do anything haha. 

I think the warning you mean is could not render file using fallback or something like that.

It's very possible that the issue is simply caused by having so many files, but I am not sure. Let me know what you find

**Comment by KosmosisDire** [comment]
This is really strange especially since I can't reproduce it at all. Did you try exporting with only a couple files like maybe 1 or 2 and see if it works?

If we could come up with a consistent way to trigger the behavior then I could fix it. But I've exported pretty large exports probably at least 30 times and nothing has ever happened.

**Comment by KosmosisDire** [comment]
Pretty large to me would be like 130 files. But you're right that's very subjective 😂😂

I can try with more files and see if I can reproduce too

**Comment by KosmosisDire** [comment]
By 130 I mean 130 notes, rather than 130 total files.  But yeah, either way it should be able to handle it!
